### PR TITLE
Add initial jq-based templating engine

### PR DESCRIPTION
**Pull request by tianon**
See also docker-library/php#1052 (and linked PRs).

This moves us from 11 templates down to 2, bringing much more consistency to the generated results.

This also adds support for Alpine 3.13 (because we might as well).

### CODE REVIEW

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
$ diff -u <(bashbrew cat https://github.com/docker-library-bot/official-images/raw/18c43fcf137c759eaf42d73c2d8bfa6c9641b4e9/library/openjdk) <(bashbrew cat <(./generate-stackbrew-library.sh))
--- /dev/fd/63	2021-01-21 23:29:15.767149125 -0800
+++ /dev/fd/62	2021-01-21 23:29:15.767149125 -0800
@@ -4,271 +4,283 @@
 Tags: 17-ea-6-jdk-oraclelinux8, 17-ea-6-oraclelinux8, 17-ea-jdk-oraclelinux8, 17-ea-oraclelinux8, 17-jdk-oraclelinux8, 17-oraclelinux8, 17-ea-6-jdk-oracle, 17-ea-6-oracle, 17-ea-jdk-oracle, 17-ea-oracle, 17-jdk-oracle, 17-oracle
 SharedTags: 17-ea-6-jdk, 17-ea-6, 17-ea-jdk, 17-ea, 17-jdk, 17
 Architectures: amd64, arm64v8
-GitCommit: 64557ee2ad82aeaae093ac069b7212cd4a0ea987
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 17/jdk/oraclelinux8
 
 Tags: 17-ea-6-jdk-oraclelinux7, 17-ea-6-oraclelinux7, 17-ea-jdk-oraclelinux7, 17-ea-oraclelinux7, 17-jdk-oraclelinux7, 17-oraclelinux7
 Architectures: amd64, arm64v8
-GitCommit: 64557ee2ad82aeaae093ac069b7212cd4a0ea987
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 17/jdk/oraclelinux7
 
 Tags: 17-ea-6-jdk-buster, 17-ea-6-buster, 17-ea-jdk-buster, 17-ea-buster, 17-jdk-buster, 17-buster
 Architectures: amd64, arm64v8
-GitCommit: 64557ee2ad82aeaae093ac069b7212cd4a0ea987
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 17/jdk/buster
 
 Tags: 17-ea-6-jdk-slim-buster, 17-ea-6-slim-buster, 17-ea-jdk-slim-buster, 17-ea-slim-buster, 17-jdk-slim-buster, 17-slim-buster, 17-ea-6-jdk-slim, 17-ea-6-slim, 17-ea-jdk-slim, 17-ea-slim, 17-jdk-slim, 17-slim
 Architectures: amd64, arm64v8
-GitCommit: 64557ee2ad82aeaae093ac069b7212cd4a0ea987
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 17/jdk/slim-buster
 
+Tags: 17-ea-5-jdk-alpine3.13, 17-ea-5-alpine3.13, 17-ea-jdk-alpine3.13, 17-ea-alpine3.13, 17-jdk-alpine3.13, 17-alpine3.13, 17-ea-5-jdk-alpine, 17-ea-5-alpine, 17-ea-jdk-alpine, 17-ea-alpine, 17-jdk-alpine, 17-alpine
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
+Directory: 17/jdk/alpine3.13
+
+Tags: 17-ea-5-jdk-alpine3.12, 17-ea-5-alpine3.12, 17-ea-jdk-alpine3.12, 17-ea-alpine3.12, 17-jdk-alpine3.12, 17-alpine3.12
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
+Directory: 17/jdk/alpine3.12
+
 Tags: 17-ea-6-jdk-windowsservercore-1809, 17-ea-6-windowsservercore-1809, 17-ea-jdk-windowsservercore-1809, 17-ea-windowsservercore-1809, 17-jdk-windowsservercore-1809, 17-windowsservercore-1809
 SharedTags: 17-ea-6-jdk-windowsservercore, 17-ea-6-windowsservercore, 17-ea-jdk-windowsservercore, 17-ea-windowsservercore, 17-jdk-windowsservercore, 17-windowsservercore, 17-ea-6-jdk, 17-ea-6, 17-ea-jdk, 17-ea, 17-jdk, 17
 Architectures: windows-amd64
-GitCommit: 64557ee2ad82aeaae093ac069b7212cd4a0ea987
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 17/jdk/windows/windowsservercore-1809
 Constraints: windowsservercore-1809
 
 Tags: 17-ea-6-jdk-windowsservercore-ltsc2016, 17-ea-6-windowsservercore-ltsc2016, 17-ea-jdk-windowsservercore-ltsc2016, 17-ea-windowsservercore-ltsc2016, 17-jdk-windowsservercore-ltsc2016, 17-windowsservercore-ltsc2016
 SharedTags: 17-ea-6-jdk-windowsservercore, 17-ea-6-windowsservercore, 17-ea-jdk-windowsservercore, 17-ea-windowsservercore, 17-jdk-windowsservercore, 17-windowsservercore, 17-ea-6-jdk, 17-ea-6, 17-ea-jdk, 17-ea, 17-jdk, 17
 Architectures: windows-amd64
-GitCommit: 64557ee2ad82aeaae093ac069b7212cd4a0ea987
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 17/jdk/windows/windowsservercore-ltsc2016
 Constraints: windowsservercore-ltsc2016
 
 Tags: 17-ea-6-jdk-nanoserver-1809, 17-ea-6-nanoserver-1809, 17-ea-jdk-nanoserver-1809, 17-ea-nanoserver-1809, 17-jdk-nanoserver-1809, 17-nanoserver-1809
 SharedTags: 17-ea-6-jdk-nanoserver, 17-ea-6-nanoserver, 17-ea-jdk-nanoserver, 17-ea-nanoserver, 17-jdk-nanoserver, 17-nanoserver
 Architectures: windows-amd64
-GitCommit: 64557ee2ad82aeaae093ac069b7212cd4a0ea987
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 17/jdk/windows/nanoserver-1809
 Constraints: nanoserver-1809, windowsservercore-1809
 
 Tags: 16-ea-33-jdk-oraclelinux8, 16-ea-33-oraclelinux8, 16-ea-jdk-oraclelinux8, 16-ea-oraclelinux8, 16-jdk-oraclelinux8, 16-oraclelinux8, 16-ea-33-jdk-oracle, 16-ea-33-oracle, 16-ea-jdk-oracle, 16-ea-oracle, 16-jdk-oracle, 16-oracle
 SharedTags: 16-ea-33-jdk, 16-ea-33, 16-ea-jdk, 16-ea, 16-jdk, 16
 Architectures: amd64, arm64v8
-GitCommit: 3f4ad224491abfb5ea96a80cbaf4ff7f1b8d9c66
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 16/jdk/oraclelinux8
 
 Tags: 16-ea-33-jdk-oraclelinux7, 16-ea-33-oraclelinux7, 16-ea-jdk-oraclelinux7, 16-ea-oraclelinux7, 16-jdk-oraclelinux7, 16-oraclelinux7
 Architectures: amd64, arm64v8
-GitCommit: 3f4ad224491abfb5ea96a80cbaf4ff7f1b8d9c66
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 16/jdk/oraclelinux7
 
 Tags: 16-ea-33-jdk-buster, 16-ea-33-buster, 16-ea-jdk-buster, 16-ea-buster, 16-jdk-buster, 16-buster
 Architectures: amd64, arm64v8
-GitCommit: 3f4ad224491abfb5ea96a80cbaf4ff7f1b8d9c66
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 16/jdk/buster
 
 Tags: 16-ea-33-jdk-slim-buster, 16-ea-33-slim-buster, 16-ea-jdk-slim-buster, 16-ea-slim-buster, 16-jdk-slim-buster, 16-slim-buster, 16-ea-33-jdk-slim, 16-ea-33-slim, 16-ea-jdk-slim, 16-ea-slim, 16-jdk-slim, 16-slim
 Architectures: amd64, arm64v8
-GitCommit: 3f4ad224491abfb5ea96a80cbaf4ff7f1b8d9c66
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 16/jdk/slim-buster
 
-Tags: 16-ea-32-jdk-alpine3.12, 16-ea-32-alpine3.12, 16-ea-jdk-alpine3.12, 16-ea-alpine3.12, 16-jdk-alpine3.12, 16-alpine3.12, 16-ea-32-jdk-alpine, 16-ea-32-alpine, 16-ea-jdk-alpine, 16-ea-alpine, 16-jdk-alpine, 16-alpine
-GitCommit: 7c1aa2acd512c1eba6ac0343f085975596648a0f
+Tags: 16-ea-32-jdk-alpine3.13, 16-ea-32-alpine3.13, 16-ea-jdk-alpine3.13, 16-ea-alpine3.13, 16-jdk-alpine3.13, 16-alpine3.13, 16-ea-32-jdk-alpine, 16-ea-32-alpine, 16-ea-jdk-alpine, 16-ea-alpine, 16-jdk-alpine, 16-alpine
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
+Directory: 16/jdk/alpine3.13
+
+Tags: 16-ea-32-jdk-alpine3.12, 16-ea-32-alpine3.12, 16-ea-jdk-alpine3.12, 16-ea-alpine3.12, 16-jdk-alpine3.12, 16-alpine3.12
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 16/jdk/alpine3.12
 
 Tags: 16-ea-33-jdk-windowsservercore-1809, 16-ea-33-windowsservercore-1809, 16-ea-jdk-windowsservercore-1809, 16-ea-windowsservercore-1809, 16-jdk-windowsservercore-1809, 16-windowsservercore-1809
 SharedTags: 16-ea-33-jdk-windowsservercore, 16-ea-33-windowsservercore, 16-ea-jdk-windowsservercore, 16-ea-windowsservercore, 16-jdk-windowsservercore, 16-windowsservercore, 16-ea-33-jdk, 16-ea-33, 16-ea-jdk, 16-ea, 16-jdk, 16
 Architectures: windows-amd64
-GitCommit: 3f4ad224491abfb5ea96a80cbaf4ff7f1b8d9c66
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 16/jdk/windows/windowsservercore-1809
 Constraints: windowsservercore-1809
 
 Tags: 16-ea-33-jdk-windowsservercore-ltsc2016, 16-ea-33-windowsservercore-ltsc2016, 16-ea-jdk-windowsservercore-ltsc2016, 16-ea-windowsservercore-ltsc2016, 16-jdk-windowsservercore-ltsc2016, 16-windowsservercore-ltsc2016
 SharedTags: 16-ea-33-jdk-windowsservercore, 16-ea-33-windowsservercore, 16-ea-jdk-windowsservercore, 16-ea-windowsservercore, 16-jdk-windowsservercore, 16-windowsservercore, 16-ea-33-jdk, 16-ea-33, 16-ea-jdk, 16-ea, 16-jdk, 16
 Architectures: windows-amd64
-GitCommit: 3f4ad224491abfb5ea96a80cbaf4ff7f1b8d9c66
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 16/jdk/windows/windowsservercore-ltsc2016
 Constraints: windowsservercore-ltsc2016
 
 Tags: 16-ea-33-jdk-nanoserver-1809, 16-ea-33-nanoserver-1809, 16-ea-jdk-nanoserver-1809, 16-ea-nanoserver-1809, 16-jdk-nanoserver-1809, 16-nanoserver-1809
 SharedTags: 16-ea-33-jdk-nanoserver, 16-ea-33-nanoserver, 16-ea-jdk-nanoserver, 16-ea-nanoserver, 16-jdk-nanoserver, 16-nanoserver
 Architectures: windows-amd64
-GitCommit: 3f4ad224491abfb5ea96a80cbaf4ff7f1b8d9c66
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 16/jdk/windows/nanoserver-1809
 Constraints: nanoserver-1809, windowsservercore-1809
 
 Tags: 15.0.2-jdk-oraclelinux8, 15.0.2-oraclelinux8, 15.0-jdk-oraclelinux8, 15.0-oraclelinux8, 15-jdk-oraclelinux8, 15-oraclelinux8, jdk-oraclelinux8, oraclelinux8, 15.0.2-jdk-oracle, 15.0.2-oracle, 15.0-jdk-oracle, 15.0-oracle, 15-jdk-oracle, 15-oracle, jdk-oracle, oracle
 SharedTags: 15.0.2-jdk, 15.0.2, 15.0-jdk, 15.0, 15-jdk, 15, jdk, latest
 Architectures: amd64, arm64v8
-GitCommit: a6515ca1ad79b6b886bb69ebe80374b61202addb
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 15/jdk/oraclelinux8
 
 Tags: 15.0.2-jdk-oraclelinux7, 15.0.2-oraclelinux7, 15.0-jdk-oraclelinux7, 15.0-oraclelinux7, 15-jdk-oraclelinux7, 15-oraclelinux7, jdk-oraclelinux7, oraclelinux7
 Architectures: amd64, arm64v8
-GitCommit: a6515ca1ad79b6b886bb69ebe80374b61202addb
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 15/jdk/oraclelinux7
 
 Tags: 15.0.2-jdk-buster, 15.0.2-buster, 15.0-jdk-buster, 15.0-buster, 15-jdk-buster, 15-buster, jdk-buster, buster
 Architectures: amd64, arm64v8
-GitCommit: a6515ca1ad79b6b886bb69ebe80374b61202addb
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 15/jdk/buster
 
 Tags: 15.0.2-jdk-slim-buster, 15.0.2-slim-buster, 15.0-jdk-slim-buster, 15.0-slim-buster, 15-jdk-slim-buster, 15-slim-buster, jdk-slim-buster, slim-buster, 15.0.2-jdk-slim, 15.0.2-slim, 15.0-jdk-slim, 15.0-slim, 15-jdk-slim, 15-slim, jdk-slim, slim
 Architectures: amd64, arm64v8
-GitCommit: a6515ca1ad79b6b886bb69ebe80374b61202addb
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 15/jdk/slim-buster
 
 Tags: 15.0.2-jdk-windowsservercore-1809, 15.0.2-windowsservercore-1809, 15.0-jdk-windowsservercore-1809, 15.0-windowsservercore-1809, 15-jdk-windowsservercore-1809, 15-windowsservercore-1809, jdk-windowsservercore-1809, windowsservercore-1809
 SharedTags: 15.0.2-jdk-windowsservercore, 15.0.2-windowsservercore, 15.0-jdk-windowsservercore, 15.0-windowsservercore, 15-jdk-windowsservercore, 15-windowsservercore, jdk-windowsservercore, windowsservercore, 15.0.2-jdk, 15.0.2, 15.0-jdk, 15.0, 15-jdk, 15, jdk, latest
 Architectures: windows-amd64
-GitCommit: a6515ca1ad79b6b886bb69ebe80374b61202addb
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 15/jdk/windows/windowsservercore-1809
 Constraints: windowsservercore-1809
 
 Tags: 15.0.2-jdk-windowsservercore-ltsc2016, 15.0.2-windowsservercore-ltsc2016, 15.0-jdk-windowsservercore-ltsc2016, 15.0-windowsservercore-ltsc2016, 15-jdk-windowsservercore-ltsc2016, 15-windowsservercore-ltsc2016, jdk-windowsservercore-ltsc2016, windowsservercore-ltsc2016
 SharedTags: 15.0.2-jdk-windowsservercore, 15.0.2-windowsservercore, 15.0-jdk-windowsservercore, 15.0-windowsservercore, 15-jdk-windowsservercore, 15-windowsservercore, jdk-windowsservercore, windowsservercore, 15.0.2-jdk, 15.0.2, 15.0-jdk, 15.0, 15-jdk, 15, jdk, latest
 Architectures: windows-amd64
-GitCommit: a6515ca1ad79b6b886bb69ebe80374b61202addb
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 15/jdk/windows/windowsservercore-ltsc2016
 Constraints: windowsservercore-ltsc2016
 
 Tags: 15.0.2-jdk-nanoserver-1809, 15.0.2-nanoserver-1809, 15.0-jdk-nanoserver-1809, 15.0-nanoserver-1809, 15-jdk-nanoserver-1809, 15-nanoserver-1809, jdk-nanoserver-1809, nanoserver-1809
 SharedTags: 15.0.2-jdk-nanoserver, 15.0.2-nanoserver, 15.0-jdk-nanoserver, 15.0-nanoserver, 15-jdk-nanoserver, 15-nanoserver, jdk-nanoserver, nanoserver
 Architectures: windows-amd64
-GitCommit: a6515ca1ad79b6b886bb69ebe80374b61202addb
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 15/jdk/windows/nanoserver-1809
 Constraints: nanoserver-1809, windowsservercore-1809
 
 Tags: 11.0.10-jdk-oraclelinux8, 11.0.10-oraclelinux8, 11.0-jdk-oraclelinux8, 11.0-oraclelinux8, 11-jdk-oraclelinux8, 11-oraclelinux8, 11.0.10-jdk-oracle, 11.0.10-oracle, 11.0-jdk-oracle, 11.0-oracle, 11-jdk-oracle, 11-oracle
 Architectures: amd64, arm64v8
-GitCommit: 69023d13a209c206c1e676b3f97843a0ee208402
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 11/jdk/oraclelinux8
 
 Tags: 11.0.10-jdk-oraclelinux7, 11.0.10-oraclelinux7, 11.0-jdk-oraclelinux7, 11.0-oraclelinux7, 11-jdk-oraclelinux7, 11-oraclelinux7
 Architectures: amd64, arm64v8
-GitCommit: 69023d13a209c206c1e676b3f97843a0ee208402
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 11/jdk/oraclelinux7
 
 Tags: 11.0.10-jdk-buster, 11.0.10-buster, 11.0-jdk-buster, 11.0-buster, 11-jdk-buster, 11-buster
 SharedTags: 11.0.10-jdk, 11.0.10, 11.0-jdk, 11.0, 11-jdk, 11
 Architectures: amd64, arm64v8
-GitCommit: 69023d13a209c206c1e676b3f97843a0ee208402
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 11/jdk/buster
 
 Tags: 11.0.10-jdk-slim-buster, 11.0.10-slim-buster, 11.0-jdk-slim-buster, 11.0-slim-buster, 11-jdk-slim-buster, 11-slim-buster, 11.0.10-jdk-slim, 11.0.10-slim, 11.0-jdk-slim, 11.0-slim, 11-jdk-slim, 11-slim
 Architectures: amd64, arm64v8
-GitCommit: 69023d13a209c206c1e676b3f97843a0ee208402
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 11/jdk/slim-buster
 
 Tags: 11.0.10-jdk-windowsservercore-1809, 11.0.10-windowsservercore-1809, 11.0-jdk-windowsservercore-1809, 11.0-windowsservercore-1809, 11-jdk-windowsservercore-1809, 11-windowsservercore-1809
 SharedTags: 11.0.10-jdk-windowsservercore, 11.0.10-windowsservercore, 11.0-jdk-windowsservercore, 11.0-windowsservercore, 11-jdk-windowsservercore, 11-windowsservercore, 11.0.10-jdk, 11.0.10, 11.0-jdk, 11.0, 11-jdk, 11
 Architectures: windows-amd64
-GitCommit: c464770f5765a0bc8b7498a8683bc6c9144796da
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 11/jdk/windows/windowsservercore-1809
 Constraints: windowsservercore-1809
 
 Tags: 11.0.10-jdk-windowsservercore-ltsc2016, 11.0.10-windowsservercore-ltsc2016, 11.0-jdk-windowsservercore-ltsc2016, 11.0-windowsservercore-ltsc2016, 11-jdk-windowsservercore-ltsc2016, 11-windowsservercore-ltsc2016
 SharedTags: 11.0.10-jdk-windowsservercore, 11.0.10-windowsservercore, 11.0-jdk-windowsservercore, 11.0-windowsservercore, 11-jdk-windowsservercore, 11-windowsservercore, 11.0.10-jdk, 11.0.10, 11.0-jdk, 11.0, 11-jdk, 11
 Architectures: windows-amd64
-GitCommit: c464770f5765a0bc8b7498a8683bc6c9144796da
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 11/jdk/windows/windowsservercore-ltsc2016
 Constraints: windowsservercore-ltsc2016
 
 Tags: 11.0.10-jdk-nanoserver-1809, 11.0.10-nanoserver-1809, 11.0-jdk-nanoserver-1809, 11.0-nanoserver-1809, 11-jdk-nanoserver-1809, 11-nanoserver-1809
 SharedTags: 11.0.10-jdk-nanoserver, 11.0.10-nanoserver, 11.0-jdk-nanoserver, 11.0-nanoserver, 11-jdk-nanoserver, 11-nanoserver
 Architectures: windows-amd64
-GitCommit: c464770f5765a0bc8b7498a8683bc6c9144796da
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 11/jdk/windows/nanoserver-1809
 Constraints: nanoserver-1809, windowsservercore-1809
 
 Tags: 11.0.10-jre-buster, 11.0-jre-buster, 11-jre-buster
 SharedTags: 11.0.10-jre, 11.0-jre, 11-jre
 Architectures: amd64, arm64v8
-GitCommit: c464770f5765a0bc8b7498a8683bc6c9144796da
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 11/jre/buster
 
 Tags: 11.0.10-jre-slim-buster, 11.0-jre-slim-buster, 11-jre-slim-buster, 11.0.10-jre-slim, 11.0-jre-slim, 11-jre-slim
 Architectures: amd64, arm64v8
-GitCommit: c464770f5765a0bc8b7498a8683bc6c9144796da
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 11/jre/slim-buster
 
 Tags: 11.0.10-jre-windowsservercore-1809, 11.0-jre-windowsservercore-1809, 11-jre-windowsservercore-1809
 SharedTags: 11.0.10-jre-windowsservercore, 11.0-jre-windowsservercore, 11-jre-windowsservercore, 11.0.10-jre, 11.0-jre, 11-jre
 Architectures: windows-amd64
-GitCommit: c464770f5765a0bc8b7498a8683bc6c9144796da
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 11/jre/windows/windowsservercore-1809
 Constraints: windowsservercore-1809
 
 Tags: 11.0.10-jre-windowsservercore-ltsc2016, 11.0-jre-windowsservercore-ltsc2016, 11-jre-windowsservercore-ltsc2016
 SharedTags: 11.0.10-jre-windowsservercore, 11.0-jre-windowsservercore, 11-jre-windowsservercore, 11.0.10-jre, 11.0-jre, 11-jre
 Architectures: windows-amd64
-GitCommit: c464770f5765a0bc8b7498a8683bc6c9144796da
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 11/jre/windows/windowsservercore-ltsc2016
 Constraints: windowsservercore-ltsc2016
 
 Tags: 11.0.10-jre-nanoserver-1809, 11.0-jre-nanoserver-1809, 11-jre-nanoserver-1809
 SharedTags: 11.0.10-jre-nanoserver, 11.0-jre-nanoserver, 11-jre-nanoserver
 Architectures: windows-amd64
-GitCommit: c464770f5765a0bc8b7498a8683bc6c9144796da
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 11/jre/windows/nanoserver-1809
 Constraints: nanoserver-1809, windowsservercore-1809
 
 Tags: 8u282-jdk-oraclelinux8, 8u282-oraclelinux8, 8-jdk-oraclelinux8, 8-oraclelinux8, 8u282-jdk-oracle, 8u282-oracle, 8-jdk-oracle, 8-oracle
-GitCommit: 7255641da223700b33e8a097bda831fee23dd62c
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 8/jdk/oraclelinux8
 
 Tags: 8u282-jdk-oraclelinux7, 8u282-oraclelinux7, 8-jdk-oraclelinux7, 8-oraclelinux7
-GitCommit: 7255641da223700b33e8a097bda831fee23dd62c
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 8/jdk/oraclelinux7
 
 Tags: 8u282-jdk-buster, 8u282-buster, 8-jdk-buster, 8-buster
 SharedTags: 8u282-jdk, 8u282, 8-jdk, 8
-GitCommit: 7255641da223700b33e8a097bda831fee23dd62c
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 8/jdk/buster
 
 Tags: 8u282-jdk-slim-buster, 8u282-slim-buster, 8-jdk-slim-buster, 8-slim-buster, 8u282-jdk-slim, 8u282-slim, 8-jdk-slim, 8-slim
-GitCommit: 7255641da223700b33e8a097bda831fee23dd62c
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 8/jdk/slim-buster
 
 Tags: 8u282-jdk-windowsservercore-1809, 8u282-windowsservercore-1809, 8-jdk-windowsservercore-1809, 8-windowsservercore-1809
 SharedTags: 8u282-jdk-windowsservercore, 8u282-windowsservercore, 8-jdk-windowsservercore, 8-windowsservercore, 8u282-jdk, 8u282, 8-jdk, 8
 Architectures: windows-amd64
-GitCommit: 7255641da223700b33e8a097bda831fee23dd62c
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 8/jdk/windows/windowsservercore-1809
 Constraints: windowsservercore-1809
 
 Tags: 8u282-jdk-windowsservercore-ltsc2016, 8u282-windowsservercore-ltsc2016, 8-jdk-windowsservercore-ltsc2016, 8-windowsservercore-ltsc2016
 SharedTags: 8u282-jdk-windowsservercore, 8u282-windowsservercore, 8-jdk-windowsservercore, 8-windowsservercore, 8u282-jdk, 8u282, 8-jdk, 8
 Architectures: windows-amd64
-GitCommit: 7255641da223700b33e8a097bda831fee23dd62c
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 8/jdk/windows/windowsservercore-ltsc2016
 Constraints: windowsservercore-ltsc2016
 
 Tags: 8u282-jdk-nanoserver-1809, 8u282-nanoserver-1809, 8-jdk-nanoserver-1809, 8-nanoserver-1809
 SharedTags: 8u282-jdk-nanoserver, 8u282-nanoserver, 8-jdk-nanoserver, 8-nanoserver
 Architectures: windows-amd64
-GitCommit: 7255641da223700b33e8a097bda831fee23dd62c
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 8/jdk/windows/nanoserver-1809
 Constraints: nanoserver-1809, windowsservercore-1809
 
 Tags: 8u282-jre-buster, 8-jre-buster
 SharedTags: 8u282-jre, 8-jre
-GitCommit: 7255641da223700b33e8a097bda831fee23dd62c
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 8/jre/buster
 
 Tags: 8u282-jre-slim-buster, 8-jre-slim-buster, 8u282-jre-slim, 8-jre-slim
-GitCommit: 7255641da223700b33e8a097bda831fee23dd62c
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 8/jre/slim-buster
 
 Tags: 8u282-jre-windowsservercore-1809, 8-jre-windowsservercore-1809
 SharedTags: 8u282-jre-windowsservercore, 8-jre-windowsservercore, 8u282-jre, 8-jre
 Architectures: windows-amd64
-GitCommit: 7255641da223700b33e8a097bda831fee23dd62c
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 8/jre/windows/windowsservercore-1809
 Constraints: windowsservercore-1809
 
 Tags: 8u282-jre-windowsservercore-ltsc2016, 8-jre-windowsservercore-ltsc2016
 SharedTags: 8u282-jre-windowsservercore, 8-jre-windowsservercore, 8u282-jre, 8-jre
 Architectures: windows-amd64
-GitCommit: 7255641da223700b33e8a097bda831fee23dd62c
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 8/jre/windows/windowsservercore-ltsc2016
 Constraints: windowsservercore-ltsc2016
 
 Tags: 8u282-jre-nanoserver-1809, 8-jre-nanoserver-1809
 SharedTags: 8u282-jre-nanoserver, 8-jre-nanoserver
 Architectures: windows-amd64
-GitCommit: 7255641da223700b33e8a097bda831fee23dd62c
+GitCommit: 8441cebbf54a6e0ce31b1544cf27571293720bf1
 Directory: 8/jre/windows/nanoserver-1809
 Constraints: nanoserver-1809, windowsservercore-1809
```

</details>

File: 11/jdk/buster/Dockerfile [hunk 1]
```diff
@@ -103,6 +102,10 @@ RUN set -eux; \
 	find "$JAVA_HOME/lib" -name '*.so' -exec dirname '{}' ';' | sort -u > /etc/ld.so.conf.d/docker-openjdk.conf; \
 	ldconfig; \
 	\
+# https://github.com/docker-library/openjdk/issues/212#issuecomment-420979840
+# https://openjdk.java.net/jeps/341
+	java -Xshare:dump; \
```
Comment (tianon) [inline 11/jdk/buster/Dockerfile:107]: This managed to find a few places where we were missing this little tidbit. :grimacing: :innocent: :+1:

**Comment by tianon** [comment]
(https://github.com/docker-library/openjdk/compare/e7c2d69a38043e0a3c3a953178e4eb67ef7c8ad3..3fec00485b8448f6a9a4c150ad8a770306ca6814 is just the rebase/version bumps :innocent:)

**Review by yosifkit — APPROVED** [review]
:eyes:

**Pull request merged**